packages/notifications/src/elements/content/Paragraph.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { StyledParagraph } from '../../styled';
+
+/**
+ * Used for multi-line Notification content. Supports all `<div>` props
+ */
+export const Paragraph = StyledParagraph as React.FC;
```

Comment:
This removes the ability to apply any props to the `Paragraph` tag.

Use the `forwardRef` API and include `HTMLAttributes<HTMLElement>` as the props